server/Cargo.toml

```diff
@@ -17,20 +18,34 @@ categories = [
 vendored-openssl = ['openssl/vendored']
 
 [dependencies]
-cargo-husky  = "1"
-chrono       = { version = "0.4", features = ["serde"] }
-hyper        = "0.14"
-log          = "0.4"
-openssl      = "0.10"
-prost        = "0.11"
-prost-build  = "0.11"
-prost-types  = "0.11"
-serde        = "1.0"
-serde_json   = "1.0"
-tokio        = { version = "1.20", features = ["full"] }
-tokio-util   = "0.7"
-tonic        = "0.8"
-tonic-health = "0.7"
+axum          = "0.5"
+cargo-husky   = "1"
+chrono        = { version = "0.4", features = ["serde"] }
+clap          = { version = "4.0", features = ["derive"] }
+config        = "0.13"
+duplicate     = "0.4"
+futures       = "0.3"
+hyper         = "0.14"
+lipsum        = "0.8"
+log           = "0.4"
+openssl       = "0.10"
+ordered-float = { version = "3.4", features = ["serde"] }
+prost         = "0.11"
+prost-build   = "0.11"
+prost-types   = "0.11"
+rand          = { version = "0.8" }
+serde         = "1.0"
+serde_json    = "1.0"
+tokio         = { version = "1.20", features = ["full"] }
+tokio-util    = "0.7"
+tonic         = "0.8"
+tonic-health  = "0.7"
+utoipa        = { version = "2.1", features = ["axum_extras", "chrono"] }
+uuid          = { version = "1.2", features = ["v4", "serde"] }
+
+[dependencies.svc-storage-client-grpc]
+git = "https://github.com/Arrow-air/svc-storage"
+tag = "v0.9.0-develop.15"
```

Comment:
this can be v0.10.0 now

Reply:
[4428782](https://github.com/Arrow-air/svc-assets/pull/16/commits/4428782e670f74e05fb753342bf5365399d11255)